recipe/meta.yaml

```diff
@@ -10,24 +10,23 @@ source:
   sha256: 2597ef8ec6af973b9c9e94849204f43f47bb5f46d8c65a0c7a6bdf95ad018704
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.8,<4.0
+    - poetry-core >=1.0.0
     - pip
-    - poetry <1.6
   run:
-    - python >=3.8
-    - postgrest >=0.10.8,<0.12
-    - realtime >=1.0.0
-    - gotrue >=1.0.4
-    - httpx >=0.23.0
-    - storage3 >=0.5.3,<0.7
-    - supafunc >=0.2.3
-    - python-semantic-release
```

Comment:
This one is only a dependency for development:
https://github.com/supabase-community/supabase-py/blob/v1.2.0/pyproject.toml#L34
so hopefully things work without it.